web/src/user/settings/UserSettingsSidebar.tsx

```diff
@@ -19,7 +19,8 @@ import { NavItemDescriptor } from '../../util/contributions'
 import { UserAreaRouteContext } from '../area/UserArea'
 
 export interface UserSettingsSidebarItemConditionContext {
-    user: Pick<GQL.IUser, 'viewerCanAdminister' | 'builtinAuth'>
+    user: Pick<GQL.IUser, 'id' | 'viewerCanAdminister' | 'builtinAuth'>
+    authenticatedUser: Pick<GQL.IUser, 'id' | 'siteAdmin'> | null
```

Comment:
This is non-nullable 
```suggestion
    authenticatedUser: Pick<GQL.IUser, 'id' | 'siteAdmin'>
```

Reply:
I couldn't make ESLint happy without declaring this to be nullable. It would require us to make the `authenticatedUser` non-nullable from the very top (where we tried to dig together on last Thursday). Which it clearly needs more effort and out of scope of this PR 😞 So I would leave the way it is and maybe someone from @sourcegraph/web could help own fix here? 🙇